examples/teleport-usage/go.mod

```diff
@@ -0,0 +1,7 @@
+module usage-script
+
+go 1.18
```

Comment:
nit: the dockerfile uses 1.20, was this intentional?

Reply:
Set to 1.19 now. I wanted to match the main repo for compat reasons and got it wrong. The code is fine on newer versions of Go but I don't want to set the go.mod to 1.20 since it's not a requirement for compiling Teleport.